cluster-operator/src/main/java/io/strimzi/operator/cluster/model/TopicOperator.java

```diff
@@ -278,8 +278,8 @@ public void generateCertificates(CertManager certManager, List<Secret> secrets)
                 } else {
                     log.debug("Topic Operator certificate already exists");
                     cert = new CertAndKey(
-                            decodeFromSecret(topicOperatorSecret.get(), "topic-operator.key"),
-                            decodeFromSecret(topicOperatorSecret.get(), "topic-operator.crt"));
+                            decodeFromSecret(topicOperatorSecret.get(), "entity-operator.key"),
```

Comment:
Why is this in the `TopicOperator.java` when it is generating Entity Operator keys?

Reply:
because these certificates are mounted in the TO container and the TLS sidecar as well and the TO image is common when deployed via EO. We agreed on Slack to change the cert/key name here even because it's just internal stuff.

Reply:
And what happens when someone wants to deploy only User Operator and no Topic Operator within Entity operator ... will it work?

Reply:
The UO goes through the TLS sidecar for connecting to Zookeeper and the EO TLS sidecar uses these keys.